In [ ]:
##  Ho: Weeekly operating cost are  higher than the  model suggests 
# Ha:Weeekly operating cost  are not  higher than the  model suggests

In [8]:
alpha=0.05
pop_mean=4000
alpha=0.05
n=25
x_bar=3050
pop_std =125



In [11]:
import scipy.stats as stats
z_crit=stats.norm.ppf(1-alpha/2)
z_crit

1.959963984540054

In [ ]:
## since ni less than 30 so, we will use 
## z_score=(x_bar-pop_mean)/pop_std

In [12]:
z_score=(x_bar-pop_mean)/pop_std

In [13]:
z_score

-7.6

In [14]:
z_crit

1.959963984540054

Here z_critical lie between -1.959963984540054 to 1.959963984540054 and z_score(=-7.6) wich does not lie in this range so our Ho is wrong
Hence Claim of restaurent ower about incresing in weekly operating coasts is wrong